### ISSUE TITLE
Garbage gone

### ISSUE THREAD

**Issue by Trefex** [amaximus/garbage-collection-card#55]
Hi,

Pickup was today and I clicked on the red icon, now the garbage dissapeared.
```
  - entity: sensor.green_trash
    type: 'custom:garbage-collection-card'
    icon_size: 25px
    icon_color: green
    title_size: 13px
    details_size: 14px
```

This is the config.

Can you tell me how to get it back ? I wanna see it always.

**Comment by amaximus** [comment]
Please note that version 1.13.0 added the feature of hiding the card when clicked if the collection is today or tomorrow (please see [1]).
The card will re-appear tomorrow or if hide_before is used then X days before the due day.

[1] https://github.com/amaximus/garbage-collection-card/releases/tag/1.13.0

**Comment by Trefex** [comment]
Ahh thanks a lot. Somehow it could make sense to mention in the docs that the default when clicking is 1 day hiding.

Thanks a lot 
T

**Comment by amaximus** [comment]
There is no parameter for turning off or on hiding the card when clicked. That's the normal behaviour.
The README was updated accordingly ([2]) when this feature was introduced.

[2] https://github.com/amaximus/garbage-collection-card/commit/520e58403cd673ad5a287217ced110650cd2604d

**Comment by Trefex** [comment]
Yes I was just saying to add “will be hidden until the next day”. 

That would remove any confusion on expected behavior.

**Comment by amaximus** [comment]
Ahh ok, added clarification on hiding card when clicked.

**Comment by Trefex** [comment]
Fixed in b2bd98c4d542f49470c005e7b90f3436d5b6c15a
